### ISSUE TITLE
Raise TransformerError or KeyError if column isn't in DataFrame

### ISSUE THREAD

**Issue by thomasfrederikhoeck** [andersbogsnes/ml_tooling#62]
https://github.com/andersbogsnes/ml_tooling/blob/180c351f84edd99627c681576bf484b605b576f7/src/ml_tooling/transformers.py#L27

Suggestion 1:
```python
try:
    return X[self.columns]
except TransformerError:
    cols_error = list(set(self.columns) - set(X.columns))
    raise TransformerError ("The DataFrame does not include the columns: %s" % cols_error)`
```

Suggestion 2:
```python
try:
    return X[self.columns]
except KeyError:
    cols_error = list(set(self.columns) - set(X.columns))
    raise KeyError("The DataFrame does not include the columns: %s" % cols_error)`
```

**Comment by andersbogsnes** [comment]
That seems like a good idea!

Suggestion 1 doesn't work unfortunately, as pandas raises a KeyError - you probably want something like:
```python
try:
   return X[self.columns]
except KeyError:
   cols_error = list(set(self.columns) - set(X.columns))
    raise TransformerError(f"The DataFrame does not include the columns: {cols_error}")
 ```

Have to catch a KeyError and raise a TransformerError

**Comment by thomasfrederikhoeck** [comment]
Ah, ofc. Merge request incoming.

**Comment by andersbogsnes** [comment]
closed by #70
